workspace/__lib__/xod/core/delta-time/patch.xodp

```diff
@@ -24,7 +24,7 @@
       }
     },
     {
-      "id": "SJVJkVcNM",
```

Comment:
Why is this change necessary?

Reply:
`delta-time` uses `system-time` and `delta` inside.
To make `RST` works properly we have to pulse `system-time` on `RST`.

Reply:
It's a links' ID ;)